src/room_buffer.rs

```diff
@@ -265,8 +265,8 @@ impl RoomBuffer {
                 }
             };
 
-            // The `_guard` expires here, releasing the mutex taken at the beginning of
-            // `update_typing_notice`.
+            // The `guard` expires here, releasing the mutex taken at the
```

Comment:
I was quoting the variable name here, but if you're removing the undescore, then we should remove the backticks too. :smile: 

```suggestion
            // The guard expires here, releasing the mutex taken at the
```

Reply:
The variable got renamed.

Reply:
I think we're talking about different variables, I was referring to [this one](https://github.com/poljar/weechat-matrix-rs/blob/ea4604f96893447973f076c30df0de99cca84e98/src/room_buffer.rs#L243), but it's fine.